packages/plugin-core/src/test/suite-integ/DoctorCommand.test.ts

```diff
@@ -11,7 +15,7 @@ import { DoctorCommand } from "../../commands/Doctor";
 import { ReloadIndexCommand } from "../../commands/ReloadIndex";
 import { onWSInit, setupDendronWorkspace } from "../testUtils";
 import { expect } from "../testUtilsv2";
-import { setupBeforeAfter } from "../testUtilsV3";
+import { runLegacySingleWorkspaceTest, setupBeforeAfter } from "../testUtilsV3";
 
 suite("notes", function () {
```

Comment:
this was my mistake but if you could change the name here to `DoctorComamndTest` :)